### PR TITLE
Capstone 1 project

### PR DESCRIPTION
**Pull request by Camilovelag**
Good morning/afternoon. 🌤️

This pull request is intended to complete capstone #1.

Changes made: 🔧

✔️ Tools were installed (linterns and bootstrap 5).
✔️ Index.html and about.html were created.
✔️ Webpage was styled.
✔️ Toggle menu was added.
✔️ Speaker card section was created dynamically using scripts.js
✔️ Images, logos, and icons were customized.

Currently, I have an issue with the ESlinter related to the Bootstrap 5 js files because they are reporting errors, but if you see the error report, there is nothing related to the scripts.js file (my js file) as you can see in the image below (you could check out searching my file in the ESlinter Error Report):

image

I mention this not because I want to skip this error. In fact, I'm asking for your help because I wasn't able to reach out for help at this time (people I know haven't enough experience working on that, and I could find working solutions on the internet).

Of course, I am aware this PR won't be approved, but I hope to get help from you and to know additional feedback to improve my project and be approved in the second CR.

I am expected for feedback.

Regards! 😸

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @Camilovelag, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project that has failing linters, Kindly make sure your linters are all passing before submitting the PR.

## ESLint AutoFix
- [ ] if you have a lot of ESLint line errors, I kindly suggest you fix them by running `npx eslint . --fix` on your command line, but please get used to fixing the errors line by line also because it'll help you learn best practices.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Comment by Camilovelag** [comment]
I solved the ESlinter problem using cdn links to install Bootstrap 5 in my project instead compiled files  🧰 
@topeogunleye thanks for the help 😄

File: about.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <header class=" fixed-top bg-light">
+    <div id="social-media-nav" class="d-none d-md-flex bg-dark justify-content-end align-items-center">
+      <a href="https://www.facebook.com/MotorBikeExpoOfficial/" class="" target="_blank" rel="noopener noreferrer">
+        <img class="img-ratio mx-3" src="./assets/icons/facebook-icon.png" alt="">
+      </a>
+      <a href="https://twitter.com/motorbikeexpo?lang=en" class="" target="_blank" rel="noopener noreferrer">
+        <img class="img-ratio mx-3" src="./assets/icons/twitter-icon.png" alt="">
```
Comment (rustam-tolipov) [inline about.html:20]: - [ ] [OPTIONAL] Please kindly add alt text to all images to make your website accessible

**Review by rustam-tolipov — APPROVED** [review]
## Hello @Camilovelag  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**